web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
+    }
+
+    .modal__content {
+        flex-grow: 1;
+
+        .simplebar-content {
+            box-sizing: border-box;
+            height: 100%;
+            display: flex;
```

Comment:
This works just fine as regular flow content; there's no need to set up a columnar flexbox that I can see.

Reply:
Removing flex also solved the problem of the footer (button area) appearing too tall which @alya had pointed out in the [CZO thread](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Modal.20to.20create.20polls)! Thanks for the suggestion!